src/runner.js

```diff
@@ -91,7 +91,7 @@ const runner = async (
       const excludeTagsOption = exclude_tags
         ? `--exclude-tags="${exclude_tags}"`
         : ''
-      const command = `${setupVenv} bin/busted --helper=spec/busted-ci-helper.lua -o hjtest --Xoutput "${xmlOutputFile}" ${excludeTagsOption} "${filename}"`
+      const command = `${setupVenv} bin/busted --helper=spec/busted-ci-helper.lua -o hjtest --Xoutput "${xmlOutputFolder}/${Date.now()}" ${excludeTagsOption} "${filename}"`
```

Comment:
Is `${Date.now())` a good name for the file?  `.xml` as a file extension would be useful.